rasa/model.py

```diff
@@ -128,16 +128,15 @@ def should_retrain_nlu(self) -> bool:
         return self.force_training or self.nlu
 
 
-def get_model(model_path: Text = DEFAULT_MODELS_PATH) -> TempDirectoryPath:
-    """Get a model and unpack it. Raises a `ModelNotFound` exception if
-    no model could be found at the provided path.
+def verify_model_path(model_path: Text = DEFAULT_MODELS_PATH) -> Text:
```

Comment:
I like your clean and precise change 💯 

One thing:
I find the naming `verify_model_path` a little bit confusing as this function is doing two things: verifying the path and also returning the path. How about naming it `get_local_model`?